website/src/content/blog/biome-v1-6.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Biome v1.6
+description: Partial support for Astro, Svelte and Vue, extend configuration from dependencies, formatter goodies.
+summary: |
+  Partial support for Astro, Svelte and Vue files, both CLI and LSP.
+  Now, the `extends` field is able to pick configuration files that are exported as a dependency, very useful in monorepo and big projects.
```

Comment:
```suggestion
  Now, the `extends` field is able to pick configuration files that are exported from a dependency.
```

I've removed the "very useful in monorepo and big projects" bit, because I think it's not related to monorepos or big projects. In fact, those already had an easier time of it, because they could just use relative paths :)